docs/sources/panels-visualizations/configure-data-links/index.md

```diff
@@ -164,31 +174,51 @@ You can configure data links for the following visualizations:
 
 {{< /column-list >}}
 
-## Data link variables
+## Actions
+
+Using actions, you can trigger processes like starting or shutting down a server, directly from a dashboard panel. [API settings](ref:api-settings) are configured in the **Add action** dialog box. You can also pass variables in the API editor.
+
+### Supported visualizations
+
+You can configure actions for the following visualizations:
+
+{{< column-list >}}
+
+- [Bar chart](ref:bar-chart)
+- [Candlestick](ref:candlestick)
+- [State timeline](ref:state-timeline)
+- [Status history](ref:status-history)
+- [Time series](ref:time-series)
```

Comment:
Time series at the top? 😁

Reply:
I'm going to leave this one as is because it's alphabetical on all of these configuration pages